yarn.lock

```diff
@@ -4079,6 +4079,18 @@
     tiny-invariant "^1.1.0"
     toformat "^2.0.0"
 
+"@uniswap/sdk-core@^3.0.2":
+  version "3.0.2"
```

Comment:
i'm surprised it's adding a new dep instead of upgrading the existing one?

Reply:
![image](https://user-images.githubusercontent.com/5773490/188469832-bc236262-8b2e-457c-b089-35626ae41381.png)

Reply:
yeah not sure what's happening here, but seems related.